rasa/importers/importer.py

```diff
@@ -309,6 +310,8 @@ async def _get_domain_with_e2e_actions(self) -> Domain:
                 if isinstance(event, ActionExecuted) and event.e2e_text
             ]
 
+        additional_e2e_action_names = list(set(additional_e2e_action_names))
```

Comment:
let's treat it as set from the beginning and convert to `list` in the end instead of converting back and forth.
Please also add / update a test for the case of duplicate actions (sorry that I missed that)

Reply:
Done both!